### DISCUSSION TITLE
auto update is no op because PowerShell is prevented by group policy

### DISCUSSION THREAD

**Discussion by xiaonerner0821** [tauri-apps/tauri#9852] [Q&A] [answered]
Tauri's automatic updates on Windows use PowerShell to invoke the installer, but the group policy on my computer prevents PowerShell from running. Can an option be added to invoke the installer using cmd?

**Comment by muwoo** [comment, marked as answer]
@xiaonerner0821 
Did you build the NSIS package? If so, I encountered a similar problem to you before, and later I submitted a PullRequest using ShellExcute to replace Powershell：
https://github.com/tauri-apps/tauri/pull/9818

This problem should have been solved, you can try updating to `tauri 1.6.7`

**Comment by xiaonerner0821** [comment]
Exactly what I would have expected, thanks a lot!
